.jenkins/lsu-perftests/launch_perftests.sh

```
@@ -7,71 +7,20 @@
-        echo 'Build failed'
-        configure_build_errors=1
-        exit 1
-    }
+# rm -rf "${build_dir}"
+# mkdir -p "${build_dir}"
 
-index=0
-result_files=""
+ctest -VV\
+    --output-on-failure \
+    -S ${src_dir}/.jenkins/lsu-perftests/ctest.cmake \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)